src/enc/encode.rs

```diff
@@ -721,7 +740,9 @@ fn EnsureInitialized<Alloc: BrotliAlloc>
     if (*s).params.quality == 0i32 || (*s).params.quality == 1i32 {
       lgwin = brotli_max_int(lgwin, 18i32);
     }
-    EncodeWindowBits(lgwin, s.params.large_window, &mut (*s).last_bytes_, &mut (*s).last_bytes_bits_);
+    if !(*s).params.bare_stream {
+      EncodeWindowBits(lgwin, s.params.large_window, &mut (*s).last_bytes_, &mut (*s).last_bytes_bits_);
```

Comment:
We don't need to do `&mut (*s).field` - `&mut s.field` works identically and looks nicer (again, machine generated code which isn't very natural).